homeassistant/components/rflink/binary_sensor.py

```diff
@@ -73,11 +73,11 @@ def __init__(
     def _handle_event(self, event):
         """Domain specific event handler."""
         command = event["command"]
-        if command == "on":
+        if command in ["on", "allon"]:
             self._state = True
-        elif command == "off":
+        elif command in ["off", "alloff"]:
             self._state = False
-
+            
```

Comment:
You must look at the details to see where the error is (not easy to follow):
* https://dev.azure.com/home-assistant/Core/_build/results?buildId=29989&view=logs&j=4e214fed-f310-53fa-0018-b38e7beb405b&t=6344b833-21e0-56d1-3a95-8a0fba7ad97a&l=19

There are some whitespaces that codecheck don't like.